src/tof/chopper.py

```diff
@@ -39,10 +45,10 @@ def __init__(
         close: sc.Variable,
         distance: sc.Variable,
         phase: sc.Variable,
-        direction: Literal['clockwise', 'anticlockwise'] = 'clockwise',
+        direction: Union[Clockwise, AntiClockwise] = Clockwise,
```

Comment:
Very unusual, using a union of `Literal`...

Reply:
I know, I didn't even know that it would work.
Do you have a more common suggestion?

Reply:
Is using `NewType` better?

Reply:
Btw, using `Literal` was a suggestion by Copilot... ;-)

Reply:
Why don't you use a single literal, `Literal['clockwise', 'anticlockwise']`? Or an enum?

Reply:
Yes, I went with `Enum` in the end.